README.md

```diff
@@ -293,7 +293,7 @@ could also build them yourself!_
 - [zigbee2mqtt](https://github.com/Koenkk/zigbee2mqtt) - Zigbee to MQTT bridge, get rid of your proprietary Zigbee bridges.
 
 ### DIY Projects
-
+- [Home-Assistant tutorials](https://gadget-freakz.com/) - A gadget blog with reviews and HA related DIY tutorial and reviews.!
```

Comment:
It seems like the website isn't named "Home-Assistant tutorials" (which is wrongly spelled anyways).

Again, the description should not mention Home Assistant (and not abbreviated either).

Please, read the guidelines of this project as requested before.

Reply:
Sorry for the mistakes.  This new for me, I will correct it.